core/CMakeLists.txt

```diff
@@ -817,6 +816,9 @@ if(CMAKE_VERSION VERSION_GREATER_EQUAL 3.16)
     set_property(SOURCE ${SQUISH_SOURCES} PROPERTY SKIP_UNITY_BUILD_INCLUSION TRUE)
 endif()
 
+include(GenerateExportHeader)
+generate_export_header(HSPlasma)
```

Comment:
Did you consider using `EXPORT_MACRO_NAME PLASMA_DLL` to reduce the amount of noise in the diff?

Reply:
I did strongly consider it for exactly that reason...  But a) I like `HSPLASMA_EXPORT` better, especially considering it now applies to non-DLL shared objects as well, and b) it's more consistent with the usual pattern for using this CMake module.